File: WebApplication/src/modules/student/models/student-filter.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { SearchIdentificationType } from "../enums/search-identification.enum";

export class StudentFilterDto {
  identificator: string;
  identificationType: SearchIdentificationType;

  educationalQualification: NomenclatureDto;
  educationalQualificationId: number | null;

  educationalForm: NomenclatureDto;
  educationalFormId: number | null;
}

File: StudentCredit.Application/Students/Dtos/StudentFilterDto.cs
using StudentCredit.Application.Students.Enums;

namespace StudentCredit.Application.Students.Dtos
{
    public class StudentFilterDto
    {
        public string Identificator { get; set; }
        public SearchIdentificationType IdentificationType { get; set; }

        public int? EducationalQualificationId { get; set; }
        public int? EducationalFormId { get; set; }
    }
}